codacy/templates/fluentd/lifecycle-police-job.yaml

```diff
@@ -0,0 +1,40 @@
+{{ if .Values.fluentdoperator.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ .Release.Name }}-fluentd-lifecycle-policy-creator"
+  labels:
+    {{- include "codacy.labels" . | indent 4 }}
```

Comment:
Watch out for the character limit. Maybe you need to truncate.

Reply:
Not here this is the list

Reply:
I truncated the value in the helper in the other pr. This is just the usage of the other values